inductiva/fluids/_post_processing.py

```diff
@@ -33,7 +35,9 @@ def render(self,
         """
 
         vtk_dir = os.path.join(self.sim_output_dir, "vtk")
+        movie_path = files.resolve_path(movie_path)
 
+        logging.info("Rendering movie to %s with %d fps.", movie_path, fps)
```

Comment:
Maybe say video here instead of movie for consistency.

Reply:
I will actually revert to movie, since everything contains movie.